src/js/extensions/auto-link.js

```diff
@@ -205,7 +205,7 @@ LINK_REGEXP_TEXT =
 
         createAutoLink: function (textNodes, href) {
             href = Util.ensureUrlHasProtocol(href);
-            var anchor = Util.createLink(this.document, textNodes, href),
+            var anchor = Util.createLink(this.document, textNodes, href, this.getEditorOption('targetBlank')),
```

Comment:
I was wondering why autoLink extension use `Util.createLink` instead of `Core.createLink` ? Because, `Core.createLink` already handle `targetBlank` option ([see](https://github.com/yabwe/medium-editor/blob/master/src/js/core.js#L1054-L1056)).

Any idea @patrick-webs ?